code/style.css

```diff
@@ -1,7 +1,249 @@
+* {
+	margin: 0px;
+	padding: 0px;
+	box-sizing: border-box;
+	max-width: 1350px;
+	font-family: "Manrope", sans-serif;
+
+	/* make use of how the fony should be rendered */
+	text-rendering: optimizeLegibility;
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	-webkit-font-smoothing: antialiased;
+	font-family: "Oswald", sans-serif;
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+/* Sign in form */
+header {
+
+	height: 350px;
+	/* text-align: center; */
+	margin-top: 200px;
+	margin-bottom: 0px;
+
+}
+
+/* .textWrapper {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+
+} */
+
+
+.headerH1 {
+	align-content: center; 
+	color: #fff;
+}
+
+.headerH1 {
+	text-align: center;
+}
+
+ h3 {
+	color: white;
+	text-align: center;
+}
+
+
+
+#headerWrapper {
+	height: 500px;
+}
+
+#headerWrapper video {
+	max-width: 1100px;
+	width: 100%;
+	height: 500px;
+	position: absolute;
+	top: 0;
+	left: 0;
+	z-index: -5;
+	object-fit: cover;
+}
+
+
+
+
+.wrapperSignIn {
+	width: 100%;
+	height: 550px;
+	margin: auto;
+	text-align: center;
+	box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+	overflow: hidden;
+	background-color: #5f9ea0;
+	color: white;
+}
+
+
+
+.formH1 {
+	padding: 15px;
+	font-size: 2.2rem;
+	text-align: center;
+	color: white;
+}
+
+.signInContainer {
+	display: grid;
+	justify-content: center;
+}
+
+.signInContainer input {
+	width: 500px;
+	margin: 10px;
+	padding: 15px;
+	background-color: #80b1b3;
+}
+
+input::placeholder {
+	color: white;
+	font-size: 1.7rem;
+	padding-left: 20px;
+	position: relative;
+	top: 7px;
+}
+
+.radioButtons {
+	
+	display: flex;
+	justify-content: space-around;
+	margin: 30px 20% 50px 20%;
+
+}
+
+.checkbox {
+	font-size: 1.5rem;
+}
+
+input[type="radio"] {
```

Comment:
I learned something new here regarding radio buttons and check-boxes, haven't used them yet. Thanks for that. :)